core/src/miner/mem_pool.rs

```diff
@@ -378,6 +430,44 @@ impl MemPool {
         self.remove(&invalid, &fetch_seq, current_time, current_timestamp);
     }
 
+    // Recover MemPool state from db stored data
+    pub fn recover_from_db(&mut self) {
+        match backup::recover_to_data(self.db.as_ref()) {
+            backup::RecoveredData {
+                by_hash,
+                first_seqs,
+                last_time,
+                last_timestamp,
+                next_transaction_id,
+            } => {
+                let items: Vec<MemPoolItem> = by_hash.into_iter().map(|(_, item)| item).collect();
+                //let ids: Vec<u64> = items.iter().map(|item| item.insertion_id).collect();
```

Comment:
What's it?

Reply:
I didn't delete that lines when I introduced `add_items`